themes/bootstrap3/js/common.js

```diff
@@ -239,21 +239,52 @@ var VuFind = (function VuFind() {
     return html.replace(/(<script[^>]*) nonce=["'].*?["']/ig, '$1 nonce="' + getCspNonce() + '"');
   };
 
+  function setInnerHtml(elm, html) {
+    elm.innerHTML = html;
+    Array.from(elm.querySelectorAll("script")).forEach(oldScript => {
```

Comment:
No need for `Array.from()` here. querySelectorAll returns NodeList that also has the forEach method.